packages/task/src/common/task-protocol.ts

```diff
@@ -28,15 +28,15 @@ export enum DependsOrder {
 }
 
 export enum RevealKind {
-    Always = 'always',
-    Silent = 'silent',
-    Never = 'never'
+    Always = 1,
+    Silent = 2,
+    Never = 3
```

Comment:
String values were provided within https://github.com/eclipse-theia/theia/pull/7982/files
Alvaro, could you check that the changes do not lead to the bug described in the PR description https://github.com/eclipse-theia/theia/pull/7982.

Reply:
Hi @RomanNikitenko, 
Very good catch ! I was not aware of the previous issue, 
I have left the Enums untouched to prevent compatibility issues and opted to use a maps to translate between config / DTO.
Comments are very welcome!

Reply:
Good catch on this, I tested this for regressions and it is working well for me, still getting the correct stringified values 👍